packages/contracts/src/dollar/ERC1155Ubiquity.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.16;
 import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
 import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Burnable.sol";
 import "@openzeppelin/contracts/token/ERC1155/extensions/ERC1155Pausable.sol";
-import "./core/UbiquityDollarManager.sol";
+import "./old/UbiquityDollarManager.sol";
```

Comment:
`old` is unexpected. What's wrong with `core`? We should delete old/unused code probably.